try {
  printMessage('hello');
} catch (e) {
  console.log('error:', e);
}
